proto/auth.proto

```diff
@@ -69,8 +69,46 @@ message _GenerateApiTokenRequest {
         Expires expires = 2;
     }
 
-    // session token recieved from `momento login` command
-    string session_token = 3;
+    // Aliases for categories of functionality.
+    enum CacheRole {
+      CachePermitNone = 0;
+      // Restricts access to apis that read and write data from caches: No higher level resource description or modification.
+      CacheReadWrite = 1;
+    }
+
+    // Aliases for categories of functionality.
+    enum TopicRole {
+      TopicPermitNone = 0;
```

Comment:
thought this was interesting

<img width="926" alt="Screen Shot 2023-05-16 at 4 29 46 PM" src="https://github.com/momentohq/client-protos/assets/11823378/e0282dae-b78a-4de8-a87a-38469552740d">

Reply:
Oh good catch!